subprojects/gradle/gradle-profile/src/main/kotlin/com/avito/android/gradle/metric/GradleCollector.kt

```diff
@@ -37,22 +36,13 @@ class GradleCollector(
         result.gradle?.also { cleanup(it) }
     }
 
-    override fun onOutput(output: CharSequence?) {
```

Comment:
Not needed anymore. 
We can find almost all info in build operations in a more type safe way.